cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -192,7 +194,7 @@ def start_master(etcd, tls):
         hookenv.log('Starting {0} service.'.format(service))
         host.service_start(service)
     hookenv.open_port(6443)
-    hookenv.status_set('active', 'Kubernetes master services ready.')
+    hookenv.log('Kubernetes master services ready.')
```

Comment:
but do we want this as a log? its not a problem but seems like its wasted data in the logging infra.